pkgs/development/python-modules/walrus/default.nix

```diff
@@ -30,7 +41,8 @@ buildPythonPackage rec {
   ];
 
   preCheck = ''
-    ${pkgs.redis}/bin/redis-server &
+    export WALRUS_REDIS_PORT=7777
```

Comment:
Why is this environment variable needed?

Reply:
The redis-server failed to bind on the default port on darwin, this configures the walrus test case to run against the custom port.

Reply:
Can't we just get rid of it and use `7777` just under that line?

Reply:
It's there to configure the upstream test cases: https://github.com/coleifer/walrus/blob/0.9.3/walrus/tests/base.py#L9

Reply:
That's the thing. Can you please add a comment for that? Someone might remove it by mistake.

Reply:
?

Reply:
```diff
- export WALRUS_REDIS_PORT=7777
- ${pkgs.redis}/bin/redis-server --port $WALRUS_REDIS_PORT &
+ ${pkgs.redis}/bin/redis-server &
```

Reply:
reverted